conda-recipe/conda_build_config.yaml

```
@@ -1,2 +1,18 @@
 numpy:
-    - 1.23
+    - '1.23'
```

Aligned with `project.toml` https://github.com/IntelPython/dpnp/blob/master/pyproject.toml#L12?

```suggestion
    - '1.25'
```